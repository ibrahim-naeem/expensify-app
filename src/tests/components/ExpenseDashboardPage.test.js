import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';
import toJson from 'enzyme-to-json';
import '../setupTests';


test('should render component', () => {
    
    const wrapper = shallow(<ExpenseDashboardPage />);

    expect(toJson(wrapper)).toMatchSnapshot()
})