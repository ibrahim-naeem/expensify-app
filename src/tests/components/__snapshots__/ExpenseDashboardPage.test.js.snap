// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div>
  This is from my dashboard component!
  <[object Object] />
  <[object Object] />
</div>
`;
